frontend/src/pages/organization/RoleByIDPage/components/RolePermissionsSection/OrgPermissionGatewayRow.tsx

```
@@ -0,0 +1,182 @@
+          "permissions.app-connections",
+          {
+            [OrgGatewayPermissionActions.Read]: true,
+            [OrgGatewayPermissionActions.Edit]: false,
+            [OrgGatewayPermissionActions.Create]: false,
+            [OrgGatewayPermissionActions.Delete]: false
+          },
+          { shouldDirty: true }
+        );
+        break;
+
+      case Permission.NoAccess:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)